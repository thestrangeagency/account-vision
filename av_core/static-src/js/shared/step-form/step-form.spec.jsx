import React from 'react';
import { shallow } from 'enzyme';

import StepForm from './step-form';

test('it renders', () => {
  const wrapper = shallow(<StepForm onSubmit={() => {}} />);
  expect(wrapper).toMatchSnapshot();
});
