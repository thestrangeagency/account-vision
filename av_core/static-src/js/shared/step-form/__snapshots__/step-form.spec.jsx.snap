// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<form
  className="step-form"
  noValidate={true}
  onSubmit={[Function]}
/>
`;
